packages/react/src/components/ComponentConformance.test.tsx

```diff
@@ -1,194 +0,0 @@
-import * as React from 'react';
```

Comment:
Not sure we're ready to delete this whole file? Instead it should be limited to only running the remaining tests for the components within this package I think.

Reply:
The existing ComponentConformance tests are redundant to the isConformant ones and both are currently running.

ComponentConformance tests

1.	Injects a className : **component-handles className**
2.	Contains a top-level file : **component-has-top-level-file**
3.	Top-level file contains a version import : currently commented out

Since the version test is currently commented out  (and will be re-released as **component-has-top-level-version** after this pull request ) it may be worth deleting the file for the time being.